docs/observability/aws/integrations/aws-ec2-metrics.md

```diff
@@ -4,159 +4,191 @@ title: AWS EC2 Metrics
-memory usage per instance for total and percentage memory usage, free
-memory use, buffers, cache memory, and system memory.
+The **AWS EC2 CPU (CloudWatch Metrics)** dashboard provides detailed information about EC2 CPU usage, like CPU utilization and CPU credits for burstable performance instances.
 
 Use this dashboard to:
 
-* Quickly identify if high memory utilization for an EC2 instance is potentially causing a production issue
-* Determine how memory is being used across buffers and cache memory.
+* Monitor Average CPU utilization over time for EC2.
+* Observe CPU Credits metrics (Usage and balance) over time.
+* Identify CPU Surplus Credits (Charged and Balance) over time.
 
-![3. AWS EC2 Metrics - Memory.png](/img/observability/ec2metrics3.png)
+<img src={useBaseUrl('/img/observability/AWS-EC2-Cloudwatch-Metrics-CPU.png')} alt="AWS-EC2-Cloudwatch-Metrics-CPU"/>
 
-### 4. AWS EC2 Metrics - Disk
+### 3.1. AWS EC2 EBS (CloudWatch Metrics)
 
-The **AWS EC2 Metrics - Disk** dashboard provides insights into EC2 disk
-usage per instance throughout your infrastructure, including directory,
-writes and usage, available bytes, used bytes, and byte read and write
-rates.
+The **AWS EC2 EBS (CloudWatch Metrics)** dashboard provides detailed information about EC2 EBS volumes for, Nitro-based instances based on EBS volumes read and write bytes, operations along with information on percentage of I/O and throughput credits remaining in the burst bucket.
 
 Use this dashboard to:
 
-* Quickly identify if high disk utilization for an EC2 instance is potentially causing a production issue.
-* Determine which directories have the most disk usage. 
-* Determine the performance of your storage by monitoring disk read/write rates.
+* Monitor EBS volumes read and write bytes over time.
+* Monitor EBS read and write ops over time.
+* EBS IO balance and Byte Balance % metric over time for Ec2 instances.
 
-![4. AWS EC2 Metrics - Disk.png](/img/observability/ec2metrics4.png)
+<img src={useBaseUrl('/img/observability/AWS-EC2-Cloudwatch-Metrics-EBS.png')} alt="AWS-EC2-Cloudwatch-Metrics-EBS"/>
 
-### 5. AWS EC2 Metrics - Network
+### 4.1. AWS EC2 Disk (CloudWatch Metrics)
 
-The **AWS EC2 Metrics - Network** dashboard provides insights into EC2
-network performance per instance across your infrastructure, including
-metrics for the average number of packets in and out, packet rate in and
-out, and byte rate in and out.
+The **AWS EC2 Disk (CloudWatch Metrics)** dashboard provides detailed information about a EC2 Instance Store Disk usage, based on disk read and write bytes, operations.
 
 Use this dashboard to:
 
-* Quickly identify if traffic sent and received rates for an EC2 instance is potentially causing a production issue.
-* Determine if any improvements need to be made to your AWS networking infrastructure for optimal performance.
+* Monitor instance store - Disk metrics like Disk read/write Bytes and Byte rate.
```

Comment:
Is this a reason for keeping hyphens here?

Reply:
Are you sure about the capitalization of Disk, Bytes, and Byte rate?

Reply:
- Hyphen was used to tell what we do with Monitor instance store. Updated the sentence for better readability.
- I am sure about the capitalization.

Reply:
Updated accordingly.